packages/icons/src/components/CryptoIcon.web.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { CryptoIcons } from '../iconVariants';
+
+type CryptoIconProps = {
+    type: CryptoIcons;
+    size?: CryptoIconSize;
+};
+
+const cryptoIconSizes = {
+    small: 24,
+    large: 42,
+} as const;
+
+type CryptoIconSize = keyof typeof cryptoIconSizes;
+
+// This will be used for web in the future
+export const CryptoIcon: React.FC<CryptoIconProps> = () => null;
```

Comment:
Isn't it kind of pointless to create this dummy cmp that does nothing? Why not just create it when it's needed?

Reply:
Because we don't know when people will need this new icons in web suite and since it's different team it's better to put empty components here so it will be obvious for them, that they can reuse this package.

I know we can place note somewhere but this most obvious way how to do it.